README.md

```diff
@@ -91,11 +91,15 @@ Please refer to our [coding standards](docs/coding-standards.md) for code styles
     ```
 
 ### Working with translations
+The following docs will get you started with development, include creating new
+strings to translate.  See [Translation and Localization](docs/translations.md)
+for general information on Relay localization.
+
 #### Getting the latest translations
 We use a [git submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules)
 for translated message files. The `--recurse-submodules` step of installation
 should bring the message files into your working directory already, but you may
-want also want to udpate the translations after install. The easiest way to do
+want also want to update the translations after install. The easiest way to do
 that is:
 
 * `git submodule update --remote`
```

Comment:
Unrelated, so not blocking, but this might be a good place to also [mention](https://mozilla.slack.com/archives/C013CSYEL5T/p1646304489835969) the Git config option to automatically do this, which has been working pretty well for me:

```sh
git config --global submodule.recurse true
```